src/meshapi/views/forms.py

```diff
@@ -134,6 +138,50 @@ def join_form(request: Request) -> Response:
             {"detail": "Your address could not be validated."}, status=status.HTTP_500_INTERNAL_SERVER_ERROR
         )
 
+    # Keep a blank JoinFormRequest to mutate with the info we've joined. We'll send it back
```

Comment:
Nope. We only touch Phone or Address if it was mutated by our validation. We don't do dupe checking because I believe the current opinion is that it would be dangerous to let people know that info exists (doxxing purposes etc)